ember-style-modifier/src/modifiers/style.ts

```diff
@@ -44,7 +44,7 @@ function compileStyles(
 }
 
 export interface StyleModifierSignature {
-  Element: HTMLElement;
+  Element: HTMLElement | SVGElement;
```

Comment:
@bwbuchanan proposed in #235 to fix the issue using `ElementCSSInlineStyle`. Did you tested that?

```suggestion
  Element: ElementCSSInlineStyle;
```

Reply:
Yes, that also works, I've committed it.

Reply:
Nope, actually it doesn't work, you can see the error in the CI. Ideas?

Reply:
This seems to be a limitation of the types provided by `ember-modifier`. I reproduced in TypeScript playground: https://www.typescriptlang.org/play?#code/MYGwhgzhAEBCkFNoG8BQ0PWAewHYQBcAnAV2AOyIAoEQAuaAURAQFsFcCBKFAX1X6pQkGAGEAFgEsQAE2gIAHgQ4yY8CEjSYseQqXKUa9Ji3adRAZQsBJXCEm4EFggE8WPLdswQSABwTUtFwA3OiY-LxAA

I think we should rollback to your original solution `HTMLElement | SVGElement`. Or do you now how to workaround it, @bwbuchanan?